dashboard/app/entities.go

```diff
@@ -554,6 +570,7 @@ type Job struct {
 	LastStarted time.Time `datastore:"Started"`
 	Finished    time.Time // if set, job is finished
 	TreeOrigin  bool      // whether the job is related to tree origin detection
+	CrossTree   bool      // whether the job is related to cross-tree bisections
```

Comment:
Is this used? Or you plan to use it in future in UI? It cannot be clearly inferred from other fields, right?